sympy/series/fourier.py

```diff
@@ -170,6 +189,42 @@ def truncate(self, n=3):
 
         return Add(*terms)
 
+    def sigma_approximation(self, n=3):
```

Comment:
The behaviour of that method is a bit different from `truncate` - it takes all nonzero terms of degree smaller than `n`, not first `n` nonzero ones.